Assets/Scripts/AdsAndIAP/AdsManager.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections;
+using System.Collections.Generic;
```

Comment:
Can you remove any namespaces from the class header which aren't being used by the class?